### ISSUE TITLE
Performance issues as soon as the win-capture-audio is installed and setup.

### ISSUE THREAD

**Issue by SirLanceAIot** [bozbez/win-capture-audio#62]
I am currently on windows 11 and have been using this for a few days when I noticed I have been encountering performance issues in most games. I was able to test this the best by loading up Apex legends and going into the firing range. With a fresh install of obs I had no performance stutters but as soon as I add the win-capture-audio plugin I notice my game stutter every few seconds. Hopefully there is a solution to this problem because so far it is my only complaint.

**Comment by ODesplugado** [comment]
yes i have the same problem. I have Windows 11 installed, an AMD Ryzen 5 3400G processor, and an RX 580 8GB graphics card. When installing the plugin, I lose an absurd amount of frames in several games.

**Comment by slhad** [comment]
Same here with W11 Ryzen 3600X + nv 1070ti but the whole system become irresponsive :/

**Comment by slhad** [comment]
Disabling "Hardware-Accelerated GPU Scheduling" fixed the problem, my graphic card was not powerfull enought to keep everything together, adding shuttering on shuttering.

W11 enable "Hardware-Accelerated GPU Scheduling" by default

**Comment by ODesplugado** [comment]
> Disabling "Hardware-Accelerated GPU Scheduling" fixed the problem, my graphic card was not powerfull enought to keep everything together, adding shuttering on shuttering.
> 
> W11 enable "Hardware-Accelerated GPU Scheduling" by default


My computer doesn't have this option, so I don't think that's the problem for me > [image](https://imgur.com/a/yJtKXMr)

**Comment by FQrabbit** [comment]
> 

Disabling hardware accelerated GPU scheduling will still cause problems
Win11 21H2 AMD3600X RTX3070

**Comment by bozbez** [comment]
I suspect this might have been caused by the MMCSS queues, try [v2.2.0-beta](https://github.com/bozbez/win-capture-audio/releases/tag/v2.2.0-beta).
